.github/workflows/docker-build-and-push.yml

```diff
@@ -0,0 +1,25 @@
+name: Docker Compose Actions Workflow
+run-name: Docker build & push
+
+on:
+  push:
+    branches: [CK-49-docker-image-push, develop]
```

Comment:
Shouldn't we only be pushing to docker on merges/pushes to `main`?

Reply:
I see you changed the line, but I still think it should be only on pushes to main, not develop, unless you want to push a staging/unstable build...

Reply:
Main and what about develop?

Reply:
Depends if you want to do an unstable build, but I think we should just start with considering our docker image to be "prod"

Reply:
Ill change that

Reply:
Done
Thanks for the feedback